Sources_v3/StreamChatUI/ChatChannel/ChatMessageReactionsView.swift

```diff
@@ -7,88 +7,47 @@ import UIKit
 
 // MARK: - Reaction
 
-class MessageReactionView: UIView {
-    var reaction: MessageReactionType? { didSet { updateReaction() } }
-    var madeByCurrentUser: Bool = false { didSet { updateTints() } }
+class MessageReactionView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
+    var reaction: MessageReactionType? { didSet { updateContentIfNeeded() } }
+    var madeByCurrentUser: Bool = false { didSet { updateContentIfNeeded() } }
     private let imageView: UIImageView = {
         let imageView = UIImageView()
-        imageView.contentMode = .scaleAspectFill
+        imageView.contentMode = .center
```

Comment:
We shouldn't resize any images in general. 
Because it can lead to resizing artifacts and have bad impression.
If imageview size is different from image size, it means we made mistake in layout